src/nodes/common/Custom.ts

```diff
@@ -28,7 +28,14 @@ export default class Custom extends Node {
 
   public setParsedFunction(parsedFunction?: ParsedFunction) {
     this.state.parsedFunction = parsedFunction;
-    this.updateNode();
+  }
+
+  public addInput(name: string): void {
+    this.addInputInterface(name);
+  }
+
+  public remInteface(name: string): void {
+    this.removeInterface(name);
   }
```

Comment:
These had to be added as `addInputInterface` and `removeInterface` are not public methods, rather protected

Reply:
I might be a bit picky, but I would call this method `removeInterface`, to just avoid confusion and in the end it only saves 3 letters aha. But up to you